src/store/getters/state-getters.ts

```diff
@@ -458,9 +460,19 @@ export const getCourtOrderStep = (state: StateIF): CourtOrderStepIF => {
 
 /** The custodian email. */
 export const getCustodianEmail = (state: StateIF): string => {
-  return 'hardcoded.custodian@email.com' // TODO: Change this once custodian details are available in state
+  return getCustodian(state)?.officer.email || '(Not entered)'
```

Comment:
I think the fallback should be in the UI component.

Reply:
Ah, definitely agree. Will fix!